mods/d2k/rules/defaults.yaml

```diff
@@ -153,16 +153,24 @@
 		AttackMoveCondition: attack-move
 		AssaultMoveCondition: assault-move
 
+^AutoTargetVehicleAssaultMove:
+	Inherits: ^AutoTargetGroundAssaultMove
+	AutoTargetPriority@VEHICLES:
+		ValidTargets: Vehicle
+		InvalidTargets: Infantry, Structure, Defense
+		RequiresCondition: bot-owned && (attack-move || assault-move)
+		Priority: 50
```

Comment:
Is there any specific reason why it's 50?

I just want to know the precedence. In build queue we define in intervals of 10 IIRC

Reply:
> Is there any specific reason why it's 50?

no